src/controllers/dynakube/connectioninfo/reconciler.go

```diff
@@ -92,7 +94,19 @@ func (r *Reconciler) reconcileOneAgentConnectionInfo() error {
 		return err
 	}
 
-	log.Info("oneagent connection info updated")
+	log.Info("OneAgent connection info updated")
+
+	if len(connectionInfo.Endpoints) == 0 {
+		log.Info("tenant has no endpoints", "tenant", connectionInfo.TenantUUID)
+	}
+
+	if len(connectionInfo.CommunicationHosts) == 0 {
+		log.Info("no OneAgent communication hosts received, tenant API requests not yet throttled")
+		return NoOneAgentCommunicationHostsError
+	}
```

Comment:
weird check, is it possible that `Endpoints` are empty but the `CommunicationHosts` are not ?
AFAIK the `CommunicationHosts` are just the `Endpoints` but formatted.

Reply:
TBH IDK, that's how the checks have been done previously as well, so I just kept them as good as possible. But for this issue the `CommunicationHosts` is what counts.